capabilityreceiver/cmd/migrateresources/main_integration_test.go

```diff
@@ -101,33 +104,40 @@ func Test_updateOperationResource(t *testing.T) {
 	defer historyRows.Close()
 	count := 0
 	for historyRows.Next() {
-		th.Assert(t, count <= 1, fmt.Sprintf("should only be one item in the database for this URL"))
+		th.Assert(t, count < 1, fmt.Sprintf("should only be one item in the database for this URL"))
```

Comment:
count < 1 ....  == 0?

Reply:
I ended up just deleting it based on your previous comment since there's already an assertion on line 123 that basically does the same thing